investigator/investigator/common.py

```diff
@@ -46,6 +46,18 @@ async def wait_for_limit(openshift: OpenShift, workflow_namespace: str):
         await sleep(Configuration.SLEEP_TIME)
         total_pending = openshift.workflow_manager.get_pending_workflows(workflow_namespace=workflow_namespace)
 
+async def schedule_kebechet_run_url(
+    openshift: OpenShift, repo: str, gitservice_repo_name: str, indexes: List[str]
+) -> int:
+    """Schedule all solvers."""
+    try:
+        await wait_for_limit(openshift, workflow_namespace=Configuration.THOTH_BACKEND_NAMESPACE)
+        is_scheduled = openshift.schedule_kebechet_run_url(repo, gitservice_repo_name)
```

Comment:
I cannot find `openshift.schedule_kebechet_run_url` this method in thoth-common @KPostOffice

Reply:
Never mind, we have the internal trigger feature that will create that method, for now we just update the database

Reply:
https://github.com/thoth-station/common/issues/1009

Reply:
++

Reply:
Added a TODO so we can have this in meanwhile